<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApmSoftwareConfig</class>
 <widget class="QWidget" name="ApmSoftwareConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
   <item>
    <widget class="QScrollArea" name="scrollArea_6">
     <property name="minimumSize">
      <size>
       <width>203</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>175</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>201</width>
        <height>598</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <layout class="QVBoxLayout" name="navBarLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinAndMaxSize</enum>
         </property>
         <item>
          <widget class="QPushButton" name="flightModesButton">
           <property name="minimumSize">
            <size>
             <width>175</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>175</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Flight Modes</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="geoFenceButton">
           <property name="minimumSize">
            <size>
             <width>175</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>175</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Geo Fence</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="basicPidButton">
           <property name="minimumSize">
            <size>
             <width>175</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>175</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Basic Tuning</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="arduCopterPidButton">
           <property name="minimumSize">
            <size>
             <width>175</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>175</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>APM:Copter PIDs</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="arduPlanePidButton">
           <property name="minimumSize">
            <size>
             <width>175</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>175</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>APM:Plane PIDs</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="arduRoverPidButton">
           <property name="minimumSize">
            <size>
             <width>175</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>175</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>APM:Rover PIDs</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="standardParamButton">
           <property name="minimumSize">
            <size>
             <width>175</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>175</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Standard Params</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="advancedParamButton">
           <property name="minimumSize">
            <size>
             <width>175</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>175</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Advanced Params</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="advParamListButton">
           <property name="minimumSize">
            <size>
             <width>175</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>175</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Full Parameter List</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="plannerConfigButton">
           <property name="minimumSize">
            <size>
             <width>175</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>175</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>APM Planner 2.0 Config</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="globalParamStateLabel">
           <property name="text">
            <string>Parameters</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="globalParamProgressBar">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="globalParamProgressLabel">
           <property name="text">
            <string>0/?</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
